java/src/main/java/ai/onnxruntime/OrtSession.java

```diff
@@ -1339,6 +1339,18 @@ public void addCoreML(Map<String, String> providerOptions) throws OrtException {
       addExecutionProvider(CoreMLProviderName, providerOptions);
     }
 
+    /**
+     * Adds WebGPU as an execution backend.
+     *
+     * @param providerOptions Configuration options for the WebGPU backend. Refer to the WebGPU
+     *     provider's documentation.
```

Comment:
nit: for consistency with the docs for the other functions
```suggestion
     *     execution provider's documentation.
```